ciao-controller/tables/workload_storage.csv

```diff
@@ -0,0 +1 @@
+79034317-3beb-447e-987d-4e310a8cf410,"",1,1,0,"image",73a86d7e-93c0-480e-9c41-ab42f69b7799
```

Comment:
I killed controller, restarted it and then checked the db.  I had two identical rows in the workload_storage table.